mbuild/formats/gomc_conf_writer.py

```diff
@@ -3153,6 +3403,20 @@ def __init__(
                 ):
                     self.LRC = self.input_variables_dict[key]
 
+            key = "IPC"
```

Comment:
I feel like these checks are similar enough they can be put in a for loop? 
```
for key in ["IPC", "TargetedSwapFreq", etc]:
     if <perform checks>:
        <assignment, may be self.__dict__[key] = val>
```

Reply:
I dont think they can be looped as the last line is re-setting the actual variable self.IPC.